README.md

```diff
@@ -177,6 +177,11 @@ Details on each item can be found in the
   - **libtidy** – namespace
     - **tidy** – async function
 
+## TypeScript support
+
+This library ships with a `.d.ts` for TypeScript users.
+There is no need to find in `@types/` or roll your own \<3.
```

Comment:
I'm fine to maintain the TS support part (likely to use this library in a few more projects).

Feel free to change/remove the readme if you find it inconsistent :)